### PR TITLE
Update merge and obsolete docs

### PR DESCRIPTION
**Pull request by shawntanzk**
Related to #138

### CODE REVIEW

File: docs/howto/merge-terms.md [hunk 1]
```diff
@@ -40,7 +40,10 @@ See [Daily Workflow](daily-curator-workflow.md) for creating branches and basic
    
 6. **Synchronize the reasoner** and make sure there are no terms that have identical definitions as a result of the merge. These are displayed with an 'equivalent' sign `≡` in the class hierarchy view on the left hand panel.   
 
-7. Save changes. 
+7. **Add alt_id to the winning term**.
+    - Navigate to the winning term and add the annotation hasAlternativeId with the ID (eg CL:0000000) of the losing term in the Literal tab. 
+
+8. Save changes.
```
Comment (dosumis) [inline docs/howto/merge-terms.md:47]: Add a note that any `id` annotation with value = the OBO ID of the losing term should be deleted.  Failure to do this breaks OBO<->OWL conversion.

**Review by dosumis — CHANGES_REQUESTED** [review]

**Comment by dosumis** [comment]
@shawntanzk - please add Paola as a reviewer.

**Comment by shawntanzk** [comment]
@paolaroncaglia could you do me a favour and look through this docs to make sure everything is alright (not just changes, like docs in general) Thanks!

**Comment by paolaroncaglia** [comment]
@shawntanzk 

> @paolaroncaglia could you do me a favour and look through this docs to make sure everything is alright (not just changes, like docs in general) Thanks!

Sure! The optimal way to do this would be for me to follow your merge/obsoletion SOP while actually merging/deleting real terms. How urgently do you need the docs to be reviewed? If it needs to be today, I'll just read them through. If it can wait until early next week (might be sooner, but just in case), I'll do some real edits. Cheers.

**Comment by shawntanzk** [comment]
thanks heaps @paolaroncaglia - not urgent at all, we already have a version of the docs live, just improving on it :)

**Comment by paolaroncaglia** [comment]
@shawntanzk good then, I made a note to test the guidelines at the earliest chance.

File: docs/howto/merge-terms.md [hunk 2]
```diff
@@ -40,7 +40,10 @@ See [Daily Workflow](daily-curator-workflow.md) for creating branches and basic
    
 6. **Synchronize the reasoner** and make sure there are no terms that have identical definitions as a result of the merge. These are displayed with an 'equivalent' sign `≡` in the class hierarchy view on the left hand panel.   
 
-7. Save changes. 
+7. **Add alt_id to the winning term**.
+    - Navigate to the winning term and add the annotation hasAlternativeId with the ID (eg CL:0000000) of the losing term in the Literal tab. 
+
```
Comment (nicolevasilevsky) [inline docs/howto/merge-terms.md:45]:
```suggestion
    - **Note**: any `id` annotation with value = the OBO ID of the losing term should be deleted. Failure to do this breaks OBO<->OWL conversion.
```

File: docs/howto/merge-terms.md (hunk 1 above)
Comment (nicolevasilevsky) [reply docs/howto/merge-terms.md:47]: I added this, thanks @dosumis

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**